src/views/List.jsx

```diff
@@ -1,16 +1,47 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 /** List component that displays items in a user's shopping cart  */
 export function List({ data }) {
+	const [filteredList, setFilteredList] = useState([]);
+	const [filterInput, setFilterInput] = useState('');
+
+	/* Use handler to change the state of filterInput  and convert all items to lowercase to facilitate a more thorough search */
+	const handleInput = (event) => {
+		const value = event.target.value.toLowerCase();
```

Comment:
Looks like this works well! Very minor thing, but you may also want to call `.trim()` here.